index.js

```diff
@@ -51,9 +52,18 @@ function createObject(item) {
     const myPage = new ScrapPage(browser);
     await myPage.openNewPage(URLPage);
     await myPage.fillInput(idInput, text, waitingTime);
+    console.log('Searching...');
     await myPage.clickButton(idButton, waitingTime);
+    console.log('Search successful');
     console.log('Getting data...');
-    const data = await myPage.getDataTable();
+    const expected = await myPage.expectedRows();
```

Comment:
I think this proses should be inside of the check checkData. to omit this step for the user,